core/domain/takeout_service_test.py

```diff
@@ -624,6 +671,38 @@ def test_export_data_nontrivial(self):
             'managed_topic_ids': [self.TOPIC_ID_1, self.TOPIC_ID_2]
         }
 
+        expected_voiceover_data = {
```

Comment:
add the `application` word in the variable as `voicoever` word used in multiple places in the codebase.

Reply:
Done

Reply:
Done

Reply:
Not done. Please only publish responses to comments once the code changes have been pushed.

Reply:
Oops. Done.